### PR TITLE
created tests for shopping cart repository, service and controller

### PR DESCRIPTION
**Pull request by nastasiiaKulinich**
image
image

### CODE REVIEW

File: src/test/java/com/example/bookstore/controller/ShoppingCartControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    @DisplayName(
+            "Verify that remove cart item method is working correctly"
+    )
+    void removeCartItem_CartItemId_NoContent() throws Exception {
+        mockMvc.perform(delete("/cart/items/1")
+                                .contentType(MediaType.APPLICATION_JSON)
+                )
+                .andExpect(status().isNoContent())
+                .andReturn();
+    }
```
Comment (Elena-Bruyako) [inline src/test/java/com/example/bookstore/controller/ShoppingCartControllerTest.java:229]: add tests for all endpoints and negative test cases also

File: src/test/java/com/example/bookstore/util/ShoppingCartControllerTestUtil.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        user.setId(1L);
+        user.setFirstName("John");
+        user.setLastName("Doe");
+        user.setEmail("admin.doe@example.com");
+        user.setPassword("securePassword");
+        user.setShippingAddress("123 Main St, City, Country");
+        Role role = new Role();
+        role.setId(2L);
+        role.setName(Role.RoleName.USER);
+        Role role2 = new Role();
```
Comment (Elena-Bruyako) [inline src/test/java/com/example/bookstore/util/ShoppingCartControllerTestUtil.java]: don't use numbers for naming

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
Good job! See comments

**Review by fetis-off — APPROVED** [review]
Well done!
